src/adaPets.js

```diff
@@ -25,11 +38,25 @@ const removePet = (selectedPetId) => {
     setError("You tried to remove a pet without selecting it!");
   } else {
     // Fill out as part of Wave 3.
+    axios.delete(BASE_URL + selectedPetId)
+    .then((response) => {
+      setResult(`Success! Deleted ${selectedPetId}`)
+    })
+    .catch((error) => {
+      setError(` Failed to remove ${ error }`)
+    })
   }
 };
 
 const addPet = (petInfo) => {
   // Fill out as part of Wave 4.
+  axios.post(BASE_URL + petInfo)
```

Comment:
Note from reading the test, `addPet` expects petInfo to be formatted as a nested array:
```
petInfo = {
   name: "a name"
   options: {
       optional features
   }
}
```

As such, this data structure needs to be flattened before it is passed to the axios call. One was to do this is with the code block below.
```
petInfo = petInfo.options
petInfo.name = petInfo.name
```